### PR TITLE
New version: CrystalRich.LockHunter version 3.4.3

### PR DESCRIPTION
**Pull request by DinCahill**
﻿- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/36795)

### CODE REVIEW

File: manifests/c/CrystalRich/LockHunter/3.4.3/CrystalRich.LockHunter.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Created with YamlCreate.ps1 v2.0.3 $debug=NVS0.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: CrystalRich.LockHunter
+PackageVersion: 3.4.3
```
Comment (ItzLevvie) [inline manifests/c/CrystalRich/LockHunter/3.4.3/CrystalRich.LockHunter.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 3.4.3.146
```

File: manifests/c/CrystalRich/LockHunter/3.4.3/CrystalRich.LockHunter.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.0.3 $debug=NVS0.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: CrystalRich.LockHunter
+PackageVersion: 3.4.3
```
Comment (ItzLevvie) [inline manifests/c/CrystalRich/LockHunter/3.4.3/CrystalRich.LockHunter.installer.yaml:5]:
```suggestion
PackageVersion: 3.4.3.146
```

File: manifests/c/CrystalRich/LockHunter/3.4.3/CrystalRich.LockHunter.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.3 $debug=NVS0.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: CrystalRich.LockHunter
+PackageVersion: 3.4.3
```
Comment (ItzLevvie) [inline manifests/c/CrystalRich/LockHunter/3.4.3/CrystalRich.LockHunter.yaml:5]:
```suggestion
PackageVersion: 3.4.3.146
```

Comment (ItzLevvie) [reply manifests/c/CrystalRich/LockHunter/3.4.3/CrystalRich.LockHunter.yaml:5]: This must match Control Panel.

File: manifests/c/CrystalRich/LockHunter/3.4.3/CrystalRich.LockHunter.locale.en-US.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/c/CrystalRich/LockHunter/3.4.3/CrystalRich.LockHunter.locale.en-US.yaml:5]: This must match Control Panel.

File: manifests/c/CrystalRich/LockHunter/3.4.3/CrystalRich.LockHunter.installer.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/c/CrystalRich/LockHunter/3.4.3/CrystalRich.LockHunter.installer.yaml:5]: This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**